tests/kuttl/common/assert-sample-deployment.yaml

```diff
@@ -159,6 +159,17 @@ spec:
         replicas: 0 # backend needs to be configured
       designateBackendbind9:
         replicas: 0 # backend needs to be configured
+  barbican:
+    enabled: false
```

Comment:
The current samples have Barbican enabled by default, since they don't have `enabled: false` in the YAML.  So we need to reflect that here.  Also, since Barbican is enabled by default, its condition entry needs to be added to the `status.conditions` list.

Reply:
It also needs an entry for the Barbican "expose" condition:

```
    - message: OpenStackControlPlane barbican service exposed
      reason: Ready
      status: "True"
      type: OpenStackControlPlaneExposeBarbicanReady
```

That should go after the `OpenStackControlPlaneClientReady` condition.

Reply:
Yes, per JP Jung, the current plan is to always enable Barbican.  This also means other projects that integrate with Barbican will be set up to use it by default. e.g. encrypted volumes will store keys there, etc.